test/selenium/lib/view/sign_up/view.js

```diff
@@ -49,6 +49,18 @@ SetUpView.prototype = {
         });
     },
 
+    successSignUpFromApp: function(password, confirmPassword) {
```

Comment:
Nit: If we know sign up should success, there is no need to let the possibility to enter wrong data. I would remove the `confirmPassword` variable